cmu_graphics/__init__.py

```diff
@@ -1,7 +1,13 @@
 import __main__
 import os
 
-main_directory = os.path.dirname(os.path.abspath(__main__.__file__))
+try:
+    main_directory = os.path.dirname(__main__.__file__)
+except Exception:
+    main_directory = os.path.dirname(os.path.abspath(__main__.__name__))
```

Comment:
It looks like `__main__.__name__` is always `__main__`, so `os.path.abspath(__main__.__name__)` isn't actually taking the absolute path of a file. I think `os.getcwd()` is equivalent to  `os.path.dirname(os.path.abspath(__main__.__name__))` here and cleaner.